app/lib/efile/md/md502_calculator.rb

```diff
@@ -79,7 +79,7 @@ def calculate
       end
 
       def refund_or_owed_amount
-        0
+        50
```

Comment:
These are stubs anyways so I updated them to be positive so that the financial transaction element will show up.

Reply:
[dust] is there any way to note that these are stubbed possibly with a comment? or, do you feel like that's not completely necessary?

Reply:
These are pretty big functionality methods so I'm probably going to leave it alone for now. I think I'll be touching these again on my next ticket anyways.